src/frontend/apps/standalone_site/src/routes/routes.tsx

```diff
@@ -27,53 +85,55 @@ type TRoutes = {
 
 export const routes: TRoutes = {
   HomePage: {
-    label: 'Dashboard',
+    label: <FormattedMessage {...messages.menuHomePageLabel} />,
```

Comment:
you should probably use the hook [useIntl](https://formatjs.io/docs/react-intl/api#useintl-hook) in this component and use the imperative api instead of `FormattedMessage` component